catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/commands/SearchFormsManageCommand.java

```diff
@@ -0,0 +1,172 @@
+
+  private final PrintStream console = System.out;
+
+  @Option(
+    name = "--list",
+    required = false,
+    aliases = {"-l"},
+    multiValued = false,
+    description = "List all the system forms before removal"
+  )
+  protected boolean viewAll = false;
+
+  @Option(
+    name = "--remove-single",
+    required = false,
+    aliases = {"-s"},
+    multiValued = false,
+    description = "Remove a single system template by metacard ID"
+  )
+  protected String singleDeletion = null;
+
+  @Option(
+    name = "--remove-all",
+    required = false,
+    aliases = {"-r"},
+    multiValued = false,
+    description = "Removes all the system templates"
+  )
+  protected boolean massDeletion = false;
+
+  @Override
+  protected Object executeWithSubject() {
+
+    if (viewAll) {
+      printSystemTemplates();
+      return null;
+    }
+
+    if (massDeletion || !isEmpty(singleDeletion)) {
+      executeDeletion();
+      return null;
+    }
+
+    return null;
+  }
+
+  private void printSystemTemplates() {
+    for (Metacard mc : retrieveSystemMetacards()) {
+      console.println(
+          ansi()
+              .fgBrightYellow()
+              .a("Title: " + mc.getTitle())
+              .newline()
+              .fgGreen()
+              .a("\t- " + mc.getId())
+              .reset());
+    }
+  }
+
+  private void executeDeletion() {
+
+    List<Metacard> allSystemTemplates = retrieveSystemMetacards();
+    if (allSystemTemplates.isEmpty()) {
+      console.println(ansi().fgBrightRed().a("No system forms exist.").reset());
+      return;
+    }
+
+    try {
+      List<String> systemTempIds =
+          singleDeletion != null
+              ? ImmutableList.of(singleDeletion)
+              : retrieveListMetacardIds(allSystemTemplates);
+
+      List<String> deletedMetacardIds = this.executeSystemTemplateRemoval(systemTempIds);
+
+      if (deletedMetacardIds.isEmpty()) {
+        console.println(ansi().fgBrightRed().a("No system forms were deleted").reset());
+      } else {
+        for (String id : deletedMetacardIds) {
+          console.println(ansi().fgBrightYellow().a("Deleted: ").fgGreen().a(id).reset());
+        }
+      }
+    } catch (IngestException | SourceUnavailableException e) {
+      console.println(ansi().fgBrightRed().a("Error removing system forms").reset());
+      LOGGER.debug("Error removing system forms", e);
+    }
+  }
+
+  private List<String> executeSystemTemplateRemoval(List<String> metacardList)
+      throws IngestException, SourceUnavailableException {
+
+    LOGGER.debug("Preparing to removing the following system form metacards", metacardList);
+
+    DeleteResponse deleteResponse =
+        catalogFramework.delete(new DeleteRequestImpl(metacardList.toArray(new String[0])));
+
+    if (!deleteResponse.getProcessingErrors().isEmpty()) {
+      console.println(ansi().fgBrightRed().a("Error removing system forms").reset());
```

Comment:
Shouldn't print here since you're already printing an error message in the `catch` above.